app/services/manifest_token_json_converter.rb

```diff
@@ -0,0 +1,22 @@
+class ManifestTokenJsonConverter
```

Comment:
Concerns: so I learn about something else. As a Rails novice, these concerns look like mixins with slightly different syntax. Is that the case?